force-app/main/default/lwc/harvestFieldMapNav/harvestFieldMapNav.js

```diff
@@ -26,9 +26,9 @@ export default class HarvestFieldMapNav extends NavigationMixin(
     LightningElement
 ) {
     @api recordId;
-    @track error;
-    @track harvestField;
-    @track harvestFields;
```

Comment:
Is this property needed?

Reply:
harvestFields property is not needed. There are are few other such properties which are not referenced. I will review the code, refactor and push those fixes in another PR.